src/loaders/ObjectLoader.js

```diff
@@ -15,21 +15,29 @@ THREE.ObjectLoader.prototype = {
 
 	load: function ( url, onLoad, onProgress, onError ) {
 
-		if ( this.texturePath === '' ) {
+		if(typeof url !==  "string" ){
```

Comment:
In the editor when reloading the page I encountered a "Uncaught TypeError: url.lastIndexOf is not a function" for 27: "this.texturePath = url.substring( 0, url.lastIndexOf( '/' ) + 1 );
This happened because the url wasn't a string anymore, but a .json file.
The error doesn't seem to happen in the dev ( so I think we can leave it out. )